### PR TITLE
fix(AWS Deploy): Correctly identify "no updates" error during deploy

### PR DESCRIPTION
**Pull request by pgrzesik**
Closes: #9241

### CODE REVIEW

File: lib/plugins/aws/lib/updateStack.js [hunk 1]
```diff
@@ -131,15 +131,14 @@ module.exports = {
...
-      .then((cfData) => this.monitorStack('update', cfData))
-      .catch((e) => {
-        if (e.message === NO_UPDATE_MESSAGE) {
-          return BbPromise.resolve();
-        }
+    try {
+      const cfData = await this.provider.request('CloudFormation', 'updateStack', params);
+      await this.monitorStack('update', cfData);
+    } catch (e) {
+      if (!e.message.includes(NO_UPDATE_MESSAGE)) {
```
Comment (medikoo) [inline lib/plugins/aws/lib/updateStack.js:138]: I think this error may happen only on `CloudFormation.updateStack` call right?

In such case, I'd wrap with try/catch only `provider.request` invocation.

**Review by medikoo — COMMENTED** [review]
@pgrzesik Looks good, I've just proposed one improvement, that should clean up our logic

File: lib/plugins/aws/lib/updateStack.js (hunk 1 above)
Comment (pgrzesik) [reply lib/plugins/aws/lib/updateStack.js]: good catch - that's correct :+1:

**Review by medikoo — APPROVED** [review]
Looks great 👍

**Pull request merged**